MIGRATION.md

```diff
@@ -1,5 +1,11 @@
 # Upgrading from 7.x to 8.x
 
+## Removal of Client-Side health check transaction filters
+
+The SDK no longer filters out health check transactions by default. Instead, they are sent to Sentry but still dropped
+by the Sentry backend by default. You can disable dropping them in your Sentry project settings. If you still want to
```

Comment:
Should we add a note here that they are dropped by default _for projects created after X_? As I guess that is the case here...? And add a note like "If your project was created before X, you can enable the inbound filter for this at Y"?

Reply:
I think the server side inbound filter was turned on for all projects 🤔 I checked for projects in Sentry that were created way before we introduced the filter and there it's also turned on.

Reply:
actually, I'm not too sure anymore
- seems like the filter was supposed to be disabled by default first (https://github.com/getsentry/sentry/issues/46731)
- later, we changed it to enabled for new projects (https://github.com/getsentry/sentry/pull/62654)
Will keep looking

Reply:
Ok, so even the projects in my private org have the filter turned on and I definitely didn't flip any switches there. I didn't find additional PRs that would have changed the defaults in older project but I asked in slack (`#discuss-issues`) to be sure.